src/System.cc

```diff
@@ -222,6 +220,23 @@ System::System(ORBVocabulary *voc, const Camera &camParams, const OrbParameters
 
 }
 
+System::~System()
+{
+    if(mpViewer) delete mptViewer;
```

Comment:
why no test with `nullptr` on this pointer?

Reply:
well, the correct code is `delete mptViewer` as `delete NULL` or `delete nullptr` is not going to break. `delete` test for nullptr by itself, no need to add boiler code ,-)